### ISSUE TITLE
Remove ability to grab screenshot on Release

### ISSUE THREAD

**Issue by hesterbruikman** [status-im/status-react#11968]
# Problem
Both iOS and Android Status app currently includes the ability to grab screenshots. On Android this ability is disabled by default, but an option is provided to enable. On iOS there is no setting to enable/disable screenshots (only 'Hide previews' when moving between apps).

There is always the option to grab another phone or a Nightly or Develop build. However, the option to enable screenshots on Android and the ability to grab screenshots on iOS increases the threat for message senders.

## Implementation
- Design input needed to prompt a dialog when a screenshot is attempted
- iOS needs further investigation. Supposedly blocking screenshots is not possible on iOS. An alternative could be to detect, warn and possibly hide the screen the user is attempting to take a screenshot of. 

## Acceptance Criteria
- Screenshots and recordings are disabled on Android Play store/App store release
- No option is provided in settings to enable Screenshots (and/or recordings) on Android Play store/App store release

## Notes
Screenshots should be possible in PR and Nightly builds for development purposes. See #11323

**Comment by nahuhh** [comment]
I'd appreciate it if the option (on android) was left as is. 
If I want to take screenshots of my passwords, my banking info, or even to take pictures of private areas or whatever I may choose. We should be allowed to assume the risk. 

Perhaps 
A) provide a warning when enabling screenshots
B) relocate the toggle to advanced options 
C) leave it there for PR, nightly AND f-droid builds as well. 

Or

Is it possible to "allow others to screenshot chats" where, if 
1. USER A = disabled and USER B = enabled
2. USER B opens chat with USER A
3. The action of tapping to enter the chat with USER A would automatically switch the screenshot state for USER B from enabled > disabled.
4. Exiting the chat would revert the change.

The action to change state should only effect users who have manually enabled screenshots.  

2b. For group chats, ~~if any one member has screenshots disabled, screenshots are disabled for all users when the conversation is in the foreground. ~~

For group and community chats, the admin should set this.

Basically, tapping to open the chat would "set screenshot toggle to off" (if previously on).
Backing out of the chat would revert to previous manually set setting. 

Cheers
🍻

**Comment by churik** [comment]
also it wouldn't be possible to get any feedback from users who faced some visual issues in-app. 
Without screenshots, I doubt that we can help.

**Comment by churik** [comment]
Reopen is still relevant
